math/least_common_multiple.cpp

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2020 @author tjgurwara99
+ * @file
+ *
+ * A basic implementation of LCM function
+ */
+
+#include <cassert>
+#include <iostream>
+
+/**
+ * Function for finding greatest common divisor of two numbers.
+ * @params two integers x and y whose gcd we want to find.
```

Comment:
The syntax is incorrect please refer #807 and [this link](https://www.doxygen.nl/manual/docblocks.html)